servers/account-data-deleter/src/config/tables.ts

```diff
@@ -193,9 +196,10 @@ const tablesWithUserId = {
   ],
 };
 
-const userTables = []
-  .concat(tablesWithUserId.tablesWithSensitivePii)
-  .concat(tablesWithUserId.tablesWithUserIdAlone);
+const userTables: string[] = [
+  ...tablesWithUserId.tablesWithSensitivePii,
```

Comment:
90% sure this is doing the same thing.. and it doesnt give type errors.

Reply:
oh I bet the original type error is because for some reason this was constructed with `[].concat(<some_list>).concat(<another_list>) vs. `some_list.concat(<another_list>)`

Reply:
can't infer element type on an empty array